pgx/run-app.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
```

Comment:
The jenkins agent may not have go installed. Please verify this and add the installation to your script if required.